Prox/Prox/Models/Place.swift

```diff
@@ -423,7 +423,7 @@ struct OpenHours {
 
     func nextOpeningTime(forTime baseTime: Date) -> String? {
         let midnight = DateComponents(hour: 0, minute: 0)
-        let times = [baseTime] + (1...7).map { getDate(forTime: midnight, onDate: baseTime.addingTimeInterval(TimeInterval($0) * AppConstants.ONE_DAY))! }
+        let times = [baseTime] + (1...1).map { getDate(forTime: midnight, onDate: baseTime.addingTimeInterval(TimeInterval($0) * AppConstants.ONE_DAY))! }
```

Comment:
nit: Either:
1) Update the method name to make it clear the span for which it turns `nextOpeningTime` (today & tomorrow)
2) Add a comment that we hacked this to only return today and tomorrow